tensorflow/inference/docker/2.7/py3/Dockerfile.graviton.ec2.cpu

```diff
@@ -68,7 +69,7 @@ RUN ${PIP} install --no-cache-dir \
     gevent==21.1.1 \
     requests==2.25.1 \
     grpcio==1.34.1 \
-    protobuf==3.15.0 \
+    protobuf==3.18.3 \
```

Comment:
Can we use this: https://github.com/aws/deep-learning-containers/pull/2243/files#diff-446e61b61a3121f41f2ea280cd7c8a966e0fa8b151133b3c5ab5fc165b42b260R162

Reply:
Updated it to use protobuf>=4.21.6, being the latest.